addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1832,11 +1832,11 @@ export function setTagName(el, newTagName) {
     while (el.firstChild) {
         n.append(el.firstChild);
     }
-    const closestLi = el.closest('li');
+    const containerEl = el.tagName === 'LI' ? el : el.parentNode;
```

Comment:
Docstring explaining the scenario will be good!, for what case we need its parent.?